pkg/types/clusterconfig/clusterconfig.go

```diff
@@ -151,6 +152,13 @@ var UserValidation = &cr.StructValidation{
 				ConvertNilToEmpty: true,
 			},
 		},
+		{
+			StructField: "SSLCertificateARN",
+			StringPtrValidation: &cr.StringPtrValidation{
+				AllowExplicitNull: true,
+				MinLength:         20,
```

Comment:
is `MinLength` necessary, or will it fail in validation anyways if it's not a valid arn?

Reply:
I see now that the error message may be misleading (although "must be greater than 20 characters" might also be misleading, so I may still be in favor of removing it). Is there an easy regex we could check, or can arn URLs take multiple forms?